scripts/aws_deploy_stack.py

```
@@ -0,0 +1,181 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import boto.cloudformation
+import boto.exception
+import boto.s3
+import sys
+import time
+import os
+import logging
+import yaml
+
+from cfn_pyplates.cli import _find_cloudformationtemplate
+from cfn_pyplates.cli import _load_pyplate
+from cfn_pyplates.options import OptionsMapping
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    stacks_yml = os.path.abspath(
+        os.path.join(
+            os.path.dirname(__file__),
+            "../configs/cloudformation/stacks.yml"))
+    parser = argparse.ArgumentParser(description='deploy a cloudformation stack')
+    parser.add_argument('stack', type=str,
+                        help='CloudFormation stack to create or update')
+    parser.add_argument('--config', type=str,
```

type=argparse.FileType('r') maybe? So it fails early if file doesn't exist.